app/configurations/config.lahti.js

```diff
@@ -172,4 +172,58 @@ export default configMerger(walttiConfig, {
       },
     },
   },
+
+  showTicketInformation: true,
+  useTicketIcons: true,
+  ticketLink: 'https://www.lsl.fi/liput-ja-hinnat/',
+  showTicketPrice: true,
+
+  showTicketLinkOnlyWhenTesting: true,
+  settingsForFeatureTesting: {
+    walkSpeed: 0.69,
+    bikeSpeed: 8.33,
+    includeBikeSuggestions: false,
+    transferPenalty: 1600,
+  },
+
+  ticketPurchaseLink: function purchaseTicketLink(fare) {
```

Comment:
I guess this code could be placed in waltti base config. Looks identical wth jyväskylä.